java/openjdk20-openj9/Portfile

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             openjdk20-openj9
+categories       java devel
+maintainers      {breun.nl:nils @breun} openmaintainer
+platforms        darwin
```

Comment:
this is implied since MacPorts v1.7 - so this line can/should be removed

Reply:
I've updated this line to `platforms {darwin any}`, because that's actually what it should be.

By the way, there seem to be thousands of ports that set this to `darwin` explicitly, and `port lint --nitpick` doesn't mention it as an issue. If the idea is to omit lines that set implicit defaults, maybe the linter should note this?

Reply:
technically it's not wrong but we've been removing this line while updating ports. I guess the linter could warn about this but nobody has implemented this check.